### PR TITLE
Cleaned the code, fixed syntax and nasty stuff

### PR DESCRIPTION
**Pull request by ExtReMLapin**
Fixed :

* Extra white spaces

* Double/triple if conditions/statements

* Deprecated function

* Unused variables

* Variable shadowing

### CODE REVIEW

**Comment by ExtReMLapin** [comment]
I'm too lazy to cleanup the whole code, only did this :

File: Clockwork/garrysmod/gamemodes/clockwork/framework/entities/weapons/cw_hands/shared.lua [hunk 1]
```diff
@@ -82,12 +82,12 @@ function SWEP:PrimaryAttack()
 		if (Clockwork.plugin:Call("PlayerCanThrowPunch", self.Owner)) then
 			self:PlayPunchAnimation();
 			self.Owner:SetAnimation(PLAYER_ATTACK1);
-			self.Weapon:SetNextPrimaryFire(CurTime() + 0.5)
-			self.Weapon:SetNextSecondaryFire(CurTime() + 0.7)
+			self:SetNextPrimaryFire(CurTime() + 0.5)
```
Comment (ExtReMLapin) [inline Clockwork/garrysmod/gamemodes/clockwork/framework/entities/weapons/cw_hands/shared.lua]: In a SWEP, self.Weapon equals self

**Comment by sanny-io** [comment]
A lot of changes here. This should have been done more progressively to make it easier to review.

**Comment by ExtReMLapin** [comment]
reeeeeeeeeeee

**Comment by ExtReMLapin** [comment]
Ok i'll make one for each type of changes

**Pull request closed without merging**